src/Bicep.Core/Semantics/Namespaces/NamespaceProvider.cs

```diff
@@ -2,22 +2,14 @@
 // Licensed under the MIT License.
 
 using System.Collections.Immutable;
-using System.Diagnostics;
-using System.Net.Http.Headers;
-using System.Security.Cryptography;
-using Bicep.Core.Configuration;
 using Bicep.Core.Diagnostics;
-using Bicep.Core.Extensions;
-using Bicep.Core.Features;
-using Bicep.Core.Registry.Oci;
```

Comment:
Strange that our [auto-formatter](https://github.com/Azure/bicep/actions/workflows/run-formatter.yml) didn't clean this up.